### PR TITLE
Execute ./mvnw -v before getting version in milestone job

### PR DESCRIPTION
**Pull request by ebyhr**
Test result in my personal repository
Screen Shot 2021-09-14 at 13 01 15

Reference https://github.com/trinodb/trino/pull/9231#issuecomment-918757679

### CODE REVIEW

**Review by hashhar — APPROVED** [review]

**Comment by ebyhr** [comment]
Faced permission issue:
Screen Shot 2021-09-14 at 14 21 49

File: .github/workflows/milestone.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ jobs:
         uses: actions/checkout@v2
       - name: Get milestone from pom.xml
         run: |
+          ./mvnw -v
```
Comment (findepi) [inline .github/workflows/milestone.yml:17]: This download things, so may need to be retried.

**Review by findepi — COMMENTED** [review]
What's the purpose of the change? Just diagnostics?

**Comment by findepi** [comment]
> I guess we need to update the setting for GHA in https://github.com/trinodb/trino/settings/actions 

`Workflow permissions` is set to `Read and write permissions`.

**Comment by ebyhr** [comment]
> What's the purpose of the change? Just diagnostics?

@findepi  The purpose is avoiding printing the download progress of maven wrapper to stdout. I should have left a comment to yml or commit though.
Screen Shot 2021-09-14 at 11 51 39

**Comment by ebyhr** [comment]
> Workflow permissions is set to Read and write permissions.

Thank you for the confirmation. It's the expected permission. I'm suspecting the permissions for the `GITHUB_TOKEN` now. 
https://docs.github.com/en/actions/reference/authentication-in-a-workflow

**Comment by ebyhr** [comment]
Looks like the below is the cause. I'm looking for the solution.
https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/
> Pull requests from public forks are still considered a special case and will receive a read token regardless of these settings.

**Comment by ebyhr** [comment]
I'm rewriting with master `push` event with `github.repos.listPullRequestsAssociatedWithCommit()`. Hopefully, I will send PR tomorrow.

**Pull request merged**